### PR TITLE
remove some dead code and fix a bug

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/plugin/dicebag/gui/DiceBagPluginView.java [hunk 1]
```diff
@@ -153,7 +155,7 @@ public void internalFrameClosing(InternalFrameEvent e)
 					WindowConstants.DO_NOTHING_ON_CLOSE);
 			}
 			else if ((answer == JOptionPane.NO_OPTION)
-				&& (answer == JOptionPane.YES_OPTION))
```
Comment (grimreaper) [inline code/src/java/plugin/dicebag/gui/DiceBagPluginView.java]: this is the bug fixed - previously this was always false

File: code/src/java/plugin/dicebag/gui/DiceBagModel.java [hunk 2]
```diff
@@ -233,10 +215,9 @@ public double rollDie(int index)
 	 * @param expression Die expression to parse
 	 * @return The double value of the expression.
 	 */
-	public double rollDie(String expression)
+	static double rollDie(String expression)
```
Comment (thpr) [inline code/src/java/plugin/dicebag/gui/DiceBagModel.java]: This is silly.  If this is going to be converted to static, it should just directly call the static in RollingMethods - in other words, this static method should be inlined.

File: code/src/java/plugin/dicebag/gui/DiceBagPluginController.java [hunk 3]
```diff
@@ -241,8 +243,7 @@ protected void openInitialBags()
 				{
 					theModel.loadDiceBag(file);
 					noLoads = false;
-				}
-				catch (Exception e)
+				} catch (Exception e)
```
Comment (thpr) [inline code/src/java/plugin/dicebag/gui/DiceBagPluginController.java]: This is a formatting violation - braces go on their own lines

**Comment by grimreaper** [comment]
Replaced with #2319

**Pull request closed without merging**